package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fh-fhc",
   "description": "A Command Line Interface for FeedHenry",
-  "version": "3.2.0-BUILD-NUMBER",
+  "version": "3.3.0-BUILD-NUMBER",
```

Comment:
is this change actually backward compatible? Looks like we are removing some commands here.

Reply:
👍 done the change as requested. Tks a lot